references/cli/inferless-model.mdx

```diff
@@ -9,34 +9,41 @@ You can use this to manage your models.
 - `list`: List all models in the current workspace
 - `delete`: delete a model from the system.
 - `rebuild`: rebuild a model.
```

Comment:
Update the description : This deploys the new code and runtime for the model

Reply:
updated